doc/api/unicode.md

```diff
@@ -1,6 +1,6 @@
-# icu
```

Comment:
going to be switching back to using `icu` as the module name since there is already a `unicode` module on npm. @rvagg helpfully grabbed the `icu` name on npm last month so we should be safe using that.